staging/com.scoreSmart/device.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const Tp = require('thingpedia');
+const unirest = require("unirest");
+
+module.exports = class ScoreDevice extends Tp.BaseDevice {
+  constructor(engine, state) {
+    super(engine, state);
+    this.uniqueId = 'com.scoreSmart';
+    this.name = 'ScoreSmart';
+    this.description = 'Keep up with the latest in soccer';
+  }
+
+  // getting league standings, given league id
+  get_standings({ league_id }) {
+    const request = require('request');
```

Comment:
Don't put a require in the middle of a method, and also you don't need `request`. 
request is callback based which doesn't play nice with a Promise-based API. You should use the Thingpedia Http library: https://almond-static.stanford.edu/jsdoc/thingpedia/latest/modules/helpers.http.html